Automattic/woocommerce-payments
# Add order update sync webhook for platform checkout

Closes #3960 

#### Changes proposed in this Pull Request

This PR adds the webhook to sync order updates with the platform checkout.

#### Testing instructions

_Platform checkout must be enabled in the account_

**Testing that the webhook is created**
1. In the WCPay settings, disable the platform checkout.
2. Go to the webhooks admin page(`/wp-admin/admin.php?page=wc-settings&tab=advanced&section=webhooks`).
3. There should be no webhooks for the platform checkout.
4. Enable the platform checkout on the WCPay settings.
5. Visit the webhooks admin page(`/wp-admin/admin.php?page=wc-settings&tab=advanced&section=webhooks`) again.
6. The webhook should have been created.
7. Try to delete the webhook, by clicking on "Delete permanently". It should be created again.


**Testing that the webhook is deleted**
1. Try to delete the webhook created in the previous step, by clicking on "Delete permanently". It should be created again.
2. Disable the platform checkout on the WCPay settings.
3. Go to the webhooks admin page(`/wp-admin/admin.php?page=wc-settings&tab=advanced&section=webhooks`).
4. The platform checkout webhook should have been deleted.

-------------------


**Post merge**
